src/axom/primal/geometry/Vector.hpp

```diff
@@ -700,6 +763,7 @@ std::ostream& operator<<(std::ostream& os, const Vector<T, NDIMS>& vec)
 
 //------------------------------------------------------------------------------
 template <typename T, int NDIMS>
+AXOM_HOST_DEVICE
 inline Vector<T, NDIMS> Vector<T, NDIMS>::make_vector(const T& x,
                                                       const T& y,
                                                       const T& z)
```

Comment:
Side note (outside the scope of this PR):

The `make_vector` utility function was introduced before we had an initializer list constructor for a `primal::Vector`. I'm not sure if we still need the `make_vector` helper function.  (and similarly for `Point::make_point`)